config.lua.dist

```diff
@@ -114,17 +118,25 @@ boostedBossKillBonus = 3
 onlyPremiumAccount = false
 
 -- Customs
+-- NOTE: weatherRain = true, activates weather raining effects
+-- NOTE: thunderEffect = true, activates thunder effects
+-- NOTE: allConsoleLog = true, show all message logs
+-- NOTE: stashMoving = true, stow an container inside your stash
+-- NOTE: depotChest, the non-stackable items will be moved to the selected depot chest(I - XVIII).
+-- NOTE: autoBank = true, the dropped coins from monsters will be automatically deposited to your bank account.
+-- NOTE: toggleGoldPouchAllowAnything will allow players to move items or gold to gold pouch
+-- NOTE: toggleServerIsRetroPVP will make this server as retro, setting PARTY_PROTECTION and ADVANCED_SECURE_MODE to 0
+-- NOTE: toggleTravelsFree will make all travels from boat free
 weatherRain = false
 thunderEffect = false
 allConsoleLog = false
--- stashMoving = true, stow an container inside your stash
--- the non-stackable items will be moved to the selected depot chest(I - XVIII).
 stashMoving = false
 depotChest = 4
 autoLoot = false
--- autoBank = true, the dropped coins from monsters will be automatically
```

Comment:
Same here

Reply:
didn't removed, just created a NOTE.

Reply:
So you removed kkkkkk
Please add this line again like this and leave the note
`autoBank = false`